app/config/index.js

```diff
@@ -123,6 +123,11 @@ function extractYargConfig(configObject, appVersion) {
           "Use contextIsolation on the main BrowserWindow (WIP - Disabling this will break most functionality)",
         type: "boolean",
       },
+      coreSettingsComposeDisableTimestampOnCopy: {
```

Comment:
short it as most users will not really know what  coreSettingsCompose is
```suggestion
      disableTimestampOnCopy: {
```

Reply:
yes I rename all of them